script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_4.txt

```diff
@@ -61,54 +61,54 @@ I cannot figure out what is causing this feeling―――
 Before I know it, Arcueid is standing right in front of me.
 ...How pathetic. She must have noticed that I am trembling.
 <Page18>
-"Yeah, I'm scared as shit. Just the thought of that vampire getting any closer makes me want to cry."
+"Yeah, I'm scared shitless. Just the thought of that vampire getting any closer makes me want to cry."
 "But Shiki, you're not crying."
 "That's because I'm holding it in. Try to understand, idiot."
 <Page19>
-She really is insensitive. It would be nice if she could understand me even if just a little.
+She really is insensitive. It would be nice if she could understand me, even if just a little.
 <Page20>
-"........" 
+"........"
 However.
 I don't know if something is off, but it looks like Arcueid is not pleased with what I just said.
 <Page21>
-"In case you didn't know, I am also a vampire-like him."
+"In case you didn't know, I am also a vampire like him."
 "Yeah. I know. I've heard it a million times."
 <Page22>
-"And so I'm asking you, why you are not afraid of me?
-Based on what you've said so far, I thought that you see me as some disgusting creature."
+"Sooo, I'm also a vampire, why aren't you afraid of me?"
+Based on what you've said so far, I would've thought that I also fall into the monster category."//to be rechecked
```

Comment:
"Like I said, I'm also a vampire. Are you sure you aren't afraid of me?"
Based on what you've said so far, I thought you would have come to think of me as a disgusting monster."